packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -1269,6 +1269,47 @@ describe("TableComponent", () => {
 		});
 	});
 
+	describe("setCellFormatter", () => {
```

Comment:
Can you add a TODO here to move this test to the `column.component.spec.ts` once that file is created together with all the tests for the `ColumnComponent`?

Reply:
I've created this issue #1469 so we don't forget about the unit testing of the `ColumnComponent`